### ISSUE TITLE
Why is design not embracing remote as much as the engineering side of tech?

### ISSUE THREAD

**Issue by klexo** [designdetails/designdetails#614]
Hiya, this question resurfaced following the patreon shout-out in the cursor episode. All their design roles (and there were loads) were limited to NY and SF whereas engineering for the same company has multiple remote openings. Have seen this across other companies like Stripe, Pinterest etc. Why is that? Shouldn't these companies be able to offer remote roles especially with the onset of remote payroll startups solving the logistics of it? Have you found working with remote design teams subpar? ):

If this was unclear plz follow up with me on twitter @ohalexis

TL;Dr
Why is design not embracing remote as much as the engineering side of tech?

**Comment by brianlovin** [comment]
Discussed in today's episode! https://designdetails.fm/episodes/r7rcX96I
